app/javascript/react/components/ReviewForm.js

```diff
@@ -0,0 +1,106 @@
+import { post } from 'fetch-mock' //What dis
+import React, { useState } from 'react'
+
+const ReviewForm = (props) => {
+  const [getReviewData, setReviewData] = useState([])
+  const [getNewReview, setNewReview] = useState({masks_customers: false, masks_employees: false, social_distancing: false, party_size: false })
+
+  const handleInputChange = event => {
+
+    const target = event.currentTarget;
+    const value = target.type === 'checkbox' ? target.checked : Number(target.value);
+
+    setNewReview({
+      ...getNewReview,
+      [event.currentTarget.name]: value
+    })
+  }
+
+  let handleSubmit = (event) => {
+    
+    event.preventDefault()
+    let formPayload = {
+      review: getNewReview
+    }
+
+    fetch(`/api/v1/restaurants/${props.id}/reviews`, {
+      credentials: "same-origin",
+      method: "POST",
+      body: JSON.stringify(formPayload),
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json"
+      }
+  })
+
+  .then(response => {
+    if (response.ok) {
+      return response
+  } else {
+      let errorMessage = `${response.status} (${response.statusText})`,
+      error = new Error(errorMessage)
+      throw error
+    }
+  })
+
+  .then(response => response.json())
+  .then(body => {
+    setReviewData([...getReviewData, getNewReview])
+  })
```

Comment:
Currently this is setting state on the `ReviewForm`, but we will want to update state on the `RestaurantShow` so that the whole page will be re-rendered with the new review.  In that case we can move the fetch call up to the `RestaurantShow` and then pass it down through `props` to the `ReviewForm`